github/activity_events.go

```diff
@@ -89,10 +89,10 @@ func (e *Event) Payload() (payload interface{}) {
 	case "WatchEvent":
 		payload = &WatchEvent{}
 	}
-	if err := json.Unmarshal(*e.RawPayload, &payload); err != nil {
-		panic(err.Error())
+	if err = json.Unmarshal(*e.RawPayload, &payload); err != nil {
```

Comment:
nit: there is no harm in leaving this as `:=` here and it is a good pattern in Go.